src/test/java/seedu/address/logic/commands/people/PeopleFindCommandTest.java

```diff
@@ -29,10 +29,10 @@
 
     @Test
     public void equals() {
-        NameContainsKeywordsPredicate firstPredicate =
-                new NameContainsKeywordsPredicate(Collections.singletonList("first"));
-        NameContainsKeywordsPredicate secondPredicate =
-                new NameContainsKeywordsPredicate(Collections.singletonList("second"));
+        PeopleNamePredicate firstPredicate =
```

Comment:
For the predicates, will `PeopleNameContainsKeywordsPredicate` sound better, or is it too long haha. Cause `PeopleNamePredicate `might be a little unclear?